.editorconfig

```
@@ -12,7 +12,7 @@ indent_style = space
 insert_final_newline = true
 trim_trailing_whitespace = true
 
-[*.{bat,cmd,[Bb][Aa][Tt],[Cc][Mm][Dd]]
+[*.{bat,cmd,[Bb][Aa][Tt],[Cc][Mm][Dd]}]
```

There is no need for `bat,cmd` It will be concatenated from all upper, lower characters
```suggestion
[*.{[Bb][Aa][Tt],[Cc][Mm][Dd]}]
```

---

👌